string topic
string filter
---
string multicast_address
uint16 port
